libs/template/templates/default/template/{{.project_name}}/pyproject.toml.tmpl

```diff
@@ -2,7 +2,7 @@
 name = "{{.project_name}}"
 version = "0.0.1"
 authors = [{ name = "{{user_name}}" }]
-requires-python = "{{template "conservative_db_connect_python_version_spec"}}"
+requires-python = "{{template "python_version_spec"}}"
```

Comment:
Below, there is a line with `"databricks-connect>=15.4,<15.5"`. Perhaps it's worth adding a short version of the DB Connect comment from versions.tmpl?